Sources/Files.swift

```diff
@@ -974,10 +974,14 @@ public typealias ReadError = FilesError<ReadErrorReason>
 // MARK: - Private system extensions
 
 private extension FileManager {
+
     func locationExists(at path: String, kind: LocationKind) -> Bool {
         var isFolder: ObjCBool = false
 
-        guard fileExists(atPath: path, isDirectory: &isFolder) else {
+        guard let decoded = path.removingPercentEncoding else {
```

Comment:
This doesn't seem like it should be a part of Files, and could end up causing unexpected behavior when working with locations that **do** include percent encoded characters in their name. Strings should be sanitized before passed into the framework.